app/services/chatbot/chatbot-interfaces.ts

```diff
@@ -819,16 +838,28 @@ export interface IBettingCommand {
 // dictionaries
 export enum ChatbotAutopermitEnums {
   'None' = 0,
-  'Subscriber Only' = 1 << 1,
+  'Subscriber Only' = 2,
+  'Regulars Only' = 4,
+  'Regulars & Subscribers' = 6,
```

Comment:
I think it's a little more clear if you keep the bit shifting here:
```TypeScript
export enum ChatbotAutopermitEnums {
  None = 0,
  SubscriberOnly = 1 << 1,
  RegularsOnly = 1 << 2,
  RegularsAndSubscribers = SubscriberOnly | RegularsOnly,
}
```

Reply:
Wouldn't work too well in this case. ChatbotAutoPermitEnums is used in the Auto permit dropdowns in the Mod Tools section.